### PR TITLE
INFRA: Initialize Acceptance Tests

### LINKED ISSUE #25: INFRA: Initialize Acceptance Tests

### PR DESCRIPTION
**Pull request by Nighila**
Closes #25

### CODE REVIEW

File: MyPortfolio.Services.Tests.Acceptance/Brokers/MyPortfolioApiBroker.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+﻿// -----------------------------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// -----------------------------------------------------------------------
+using Microsoft.AspNetCore.Mvc.Testing;
```
Comment (ShriHumrudha) [inline MyPortfolio.Services.Tests.Acceptance/Brokers/MyPortfolioApiBroker.cs:4]: Leave a space between the comments and the using statement.

File: MyPortfolio.Services.Tests.Acceptance/MyPortfolio.Services.Tests.Acceptance.csproj [hunk 2]
```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+
```
Comment (ShriHumrudha) [inline MyPortfolio.Services.Tests.Acceptance/MyPortfolio.Services.Tests.Acceptance.csproj:5]: Use this command here to resolve the issue.

```chsarp
<DisableImplicitNamespaceImports>true</DisableImplicitNamespaceImports>
```

**Review by ShriHumrudha — APPROVED** [review]

File: MyPortfolio.Services.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+using System.Threading.Tasks;
+using FluentAssertions;
+using Xunit;
+
+namespace MyPortfolio.Services.Tests.Acceptance.Apis.Home
+{
+    public partial class HomeApiTests
+    {
+
```
Comment (NaynaMallick) [inline MyPortfolio.Services.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs:13]: Remove Extra Line

File: MyPortfolio.Services.Tests.Acceptance/Apis/Home/HomeApiTests.cs [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+using MyPortfolio.Services.Tests.Acceptance.Brokers;
+using Xunit;
+
+namespace MyPortfolio.Services.Tests.Acceptance.Apis.Home
+{
+    [Collection(nameof(ApiTestCollection))]
+    public partial class HomeApiTests
+    {
+        private readonly MyPortfolioApiBroker myportfolioApiBroker;
```
Comment (ShriHumrudha) [inline MyPortfolio.Services.Tests.Acceptance/Apis/Home/HomeApiTests.cs:13]: we will be following 'Camel casing' - my**P**ortfolioApiBroker

File: MyPortfolio.Services.Tests.Acceptance/MyPortfolio.Services.Tests.Acceptance.csproj [hunk 5]
```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+	  <DisableImplicitNamespaceImports>true</DisableImplicitNamespaceImports>
```
Comment (ShriHumrudha) [inline MyPortfolio.Services.Tests.Acceptance/MyPortfolio.Services.Tests.Acceptance.csproj:5]: Align it to the previous tag start.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

**Review by NaynaMallick — APPROVED** [review]

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**